autoload/sneak/util.vim

```diff
@@ -120,6 +121,7 @@ endfunc
 
 func! s:init_hl() abort
   exec "highlight default Sneak guifg=white guibg=magenta ctermfg=white ctermbg=".(&t_Co < 256 ? "magenta" : "201")
+  exec "highlight default SneakCurrent guifg=black guibg=LightMagenta ctermfg=0 ctermbg=LightMagenta"
```

Comment:
Please make sure this is ok,
I'm not sure what kind of terminals the `&t_Co` is for and if we need one for the LightMagenta.

Reply:
we can try without the `t_Co` check